src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -228,8 +228,8 @@ void func_8084BDFC(Player* this, PlayState* play);
 void func_8084BF1C(Player* this, PlayState* play);
 void Player_UpdateCommon(Player* this, PlayState* play, Input* input);
 void func_8084C5F8(Player* this, PlayState* play);
-void func_8084C760(Player* this, PlayState* play);
-void func_8084C81C(Player* this, PlayState* play);
+void Player_UpdateInsideCrawlspace(Player* this, PlayState* play);
+void Player_UpdateLeavingCrawlspace(Player* this, PlayState* play);
```

Comment:
these are 674 "action funcs", so not sure I am on board with naming it "UpdateX"
a comment above these functions might be better until docs pick up and we have a naming scheme

Reply:
I'm all for a `Player_Action674_` prefix

Reply:
I've left these unnamed for now, the comments above should be enough until this is worked out